portality/static/doaj/doajArticles.xsd

```diff
@@ -130,15 +130,13 @@
      </xs:simpleContent>
     </xs:complexType>
    </xs:element>
- 
-   <xs:element name="fullTextUrl">
-    <xs:complexType> 
-     <xs:simpleContent>
-      <xs:extension base="xs:anyURI">
-       <xs:attribute name="format" />  
-      </xs:extension>
-     </xs:simpleContent>
-    </xs:complexType>
+
+         <xs:element name="fullTextUrl">
+    <xs:simpleType>
+     <xs:restriction base="xs:string">
```

Comment:
I think we can add the regex to the `xsd:anyURI` type, so we get some extra validation (such as no double `#` characters according to the docs)

So we should keep the type as anyURI rather than changing it to a string.